downstream/modules/platform/con-controller-role-based-access-controls.adoc

```diff
@@ -0,0 +1,13 @@
+[id="controller-role-based-access-controls"]
+
+= Role-based access controls
+
+To edit and delete a workflow job template, you must have the administrator role. 
+To create a workflow job template, you must be an organization administrator or a system administrator. 
+However, you can run a workflow job template that contains job templates that you do not have permissions for. 
+Organization administrators can create a blank workflow and then grant an `admin_role` to a low-level user, after which they can delegate more access and build the graph. 
+You must have `execute` access to a job template to add it to a workflow job template.
+
+Other tasks such as the ability to make a duplicate copy and re-launch a workflow can also be performed, depending on what kinds of permissions are granted to a particular user. You must have permissions to all the resources used in a workflow, such as job templates, before relaunching or making a copy.
```

Comment:
You can also perform other tasks, such as making a duplicate copy or re-launching a workflow, depending on the which permissions are granted to a user. ?